addons/website_forum/models/forum_post.py

```diff
@@ -554,8 +554,6 @@ def flag(self):
         for post in self:
             if not post.can_flag:
                 raise AccessError(_('%d karma required to flag a post.', post.forum_id.karma_flag))
-            if post.state == 'flagged':
```

Comment:
shouldn't you remove the error from the JS ?

Reply:
yes it should, corrected it now :)